### DISCUSSION TITLE
deb-src repo?

### DISCUSSION THREAD

**Discussion by yurikhan** [Floorp-Projects/Floorp#847] [Q&A]
The [Download](https://floorp.app/en/download/) page for Linux offers an APT sources.list.d file that consists of a single line:

```
deb [signed-by=/usr/share/keyrings/Floorp.gpg] https://ppa.floorp.app/$(ARCH) ./
```

This allows installation of the binary package with `apt-get install` but not retrieval of the source package with `apt-get source`.

Downloading a source package is one of the easiest ways for a user to get the source, apply additional customizations and package the result in their own PPA.

So, question: What would I need to put in `/etc/apt/sources.list.d/Floorp.list` in order for `apt-get source floorp` to succeed?

**Comment by creeper-0910** [comment]
Due to the complexity of the build, no method is provided to obtain it with apt-get source.
Please download the source code from Github and build it.

**Reply by yurikhan** [reply]
Oh. It looks like I was misled by this point on the main page then:

> # Completely Open Source
>
> The source code for Floorp is fully public, allowing anyone to view it and build Floorp. Not only is the browser itself open to the public, but the build environment is also open source.

I assumed deb packaging was part of the build environment.

**Reply by creeper-0910** [reply]
Oh,sorry...
It is about github workflow.
